ts-fold.el

```diff
@@ -365,11 +362,11 @@ then return the last iterated node.
 Argument NEXT is a boolean type.  If non-nil iterate forward; otherwise iterate
 in backward direction."
   (let ((iter-node node) (last-node node)
-        (last-line (car (tsc-node-start-point node))) line text break
+        (last-line (treesit-node-start node)) line text break
         (line-range 1) (last-line-range 1) max-line-range)
     (while (and iter-node (not break))
-      (setq text (tsc-node-text iter-node)
-            line (car (tsc-node-start-point iter-node))
+      (setq text (treesit-node-text iter-node)
+            line (treesit-node-start iter-node)
```

Comment:
treesit.el doesn't seem to distinguish between points and pos, so it only has `treesit-node-start`. In testing it doesn't seem to have made a difference, but I didn't test extensively